docs/modules/ROOT/nav.adoc

```diff
@@ -11,6 +11,8 @@
 ** xref:javalib/publishing.adoc[]
 ** xref:javalib/build-examples.adoc[]
 ** xref:javalib/web-examples.adoc[]
+** xref:javalib/android-examples.adoc[]
+** xref:javalib/android-linting.adoc[]
```

Comment:
This is probably a merge conflict; we moved the stuff into the `android/` subfolder